api/api.yaml

```diff
@@ -171,6 +240,27 @@ paths:
               schema:
                 $ref: "#/components/schemas/Patient"
 
+  /web/private/network/organizations/transfer:
```

Comment:
I think `transfer` is a filter on the set of organizations right? Than I would suggest to make it a query param. And than I think you use the type as defined in the eTransfer bolt: `eOverdracht-receiver` https://nuts-foundation.gitbook.io/bolts/eoverdracht/leveranciersspecificatie#4-1-2-organisatie-endpoint-discovery which makes this reusable for other bolts.